lib/ae_page_objects/core/dsl.rb

```diff
@@ -20,7 +20,9 @@ def element(name, options = {}, &block)
 
       self.element_attributes[name.to_sym] = klass
 
-      define_method name do |&block|
+      define_method name do |&block_for_define_method|
+        raise ArgumentError, "Cannot pass block to #{name}" if block_for_define_method
+
         ElementProxy.new(klass, self, options, &block)
```

Comment:
This is wrong. Before `block` referenced the variable local to the define_method block, but now it's referencing the parameter to `element`.